shotover-proxy/src/codec/redis.rs

```diff
@@ -68,6 +79,26 @@ impl Decoder for RedisCodec {
     }
 }
 
+/// It is critical that shotover only send valid redis commands.
+/// An invalid command could result in the redis instance returning multiple responses breaking shotovers transform invariants.
+fn validate_command(frame: &Frame) -> Result<()> {
```

Comment:
This could just take a Redis frame, and we can unwrap on `into_redis` in the function that calls it.

Reply:
we cant use into_redis because into_redis consumes the Frame